tests/nts/README.md

```diff
@@ -0,0 +1,14 @@
+# New Test Suite
+
+Welcome to the *new* test suite for Salt!
+
+Any test under this directory shall be written exploring the full capabilities of PyTest.
+That means, no occurrences of the [TestCase](https://docs.python.org/3/library/unittest.html#unittest.TestCase) class
+shall be used, neither our [customizations to it](../support/case.py).
```

Comment:
Would it be possible/simple to enforce this via a test? I'd be 👍 for that.

Reply:
You mean force new tests to be on the PyTest only directory?
Once we fully switch we could make a pre-commit hook to enforce that.

Reply:
no - you say that `TestCase` and `support.case` shouldn't be used in the new directory. Can we easily enforce *that*?

Reply:
Ah! Yes we can!

Reply:
Done!